test/m1_facilitator/models/ERC20GatewayTokenPair/util.ts

```diff
@@ -0,0 +1,66 @@
+// Copyright 2019 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// ----------------------------------------------------------------------------
+
+import ERC20GatewayTokenPair from '../../../../src/m1_facilitator/models/ERC20GatewayTokenPair';
+import assert from '../../../test_utils/assert';
+
+export interface ERC20GatewayTokenPairAttributes {
+  erc20Gateway?: string;
+  valueToken?: string;
+  utilityToken?: string;
+  createdAt?: Date;
+  updatedAt?: Date;
+}
+
+export function assertERC20GatewayTokenPairAttributes(
+  erc20GatewayTokenPair: ERC20GatewayTokenPair,
+  attributes: ERC20GatewayTokenPairAttributes,
+): void {
+  if (Object.prototype.hasOwnProperty.call(attributes, 'erc20Gateway')) {
```

Comment:
Why is if condition required?
I think that it should just assert.

Reply:
1. I am confused on the first comment. Why it should be done not in the top of the file if it's for class and there is no `convertToGateway` function in this file.

2. The idea is that if one would like to assert only valueToken for example, it will specify in attributes only valueToken. That's why it's not an assertion.

Reply:
ok